smartconfig/_registry.py

```diff
@@ -0,0 +1,22 @@
+from typing import Dict
+
+import smartconfig
+from smartconfig.typehints import _EntryMappingRegister
+
+
+class Registry:
```

Comment:
A singleton class that stores globals seems redundant. You could just define the two attributes at the module's scope and access them directly.